internal/controller/mem_consistency_test.go

```diff
@@ -194,17 +207,27 @@ func TestControllerRuntimeMemoryCloning(t *testing.T) {
 
 	// Create the v1 watch list client to be returned when the controller runner
 	// calls WatchList on the v1 testing type.
-	v1WatchListClient := watchListEvents(t, &pbresource.WatchEvent{
-		Operation: pbresource.WatchEvent_OPERATION_UPSERT,
-		Resource:  res1,
-	})
+	v1WatchListClientCreate := func() pbresource.ResourceService_WatchListClient {
```

Comment:
Switched these to builders, to more easily emit a complete copy once each for the priming and standard call to WatchList.